docs/tutorials/intro/initializing_brigade.ipynb

```diff
@@ -0,0 +1,232 @@
+       ".pygments .sd { color: #BA2121; font-style: italic } /* Literal.String.Doc */\n",
+       ".pygments .s2 { color: #BA2121 } /* Literal.String.Double */\n",
+       ".pygments .se { color: #BB6622; font-weight: bold } /* Literal.String.Escape */\n",
+       ".pygments .sh { color: #BA2121 } /* Literal.String.Heredoc */\n",
+       ".pygments .si { color: #BB6688; font-weight: bold } /* Literal.String.Interpol */\n",
+       ".pygments .sx { color: #008000 } /* Literal.String.Other */\n",
+       ".pygments .sr { color: #BB6688 } /* Literal.String.Regex */\n",
+       ".pygments .s1 { color: #BA2121 } /* Literal.String.Single */\n",
+       ".pygments .ss { color: #19177C } /* Literal.String.Symbol */\n",
+       ".pygments .bp { color: #008000 } /* Name.Builtin.Pseudo */\n",
+       ".pygments .fm { color: #0000FF } /* Name.Function.Magic */\n",
+       ".pygments .vc { color: #19177C } /* Name.Variable.Class */\n",
+       ".pygments .vg { color: #19177C } /* Name.Variable.Global */\n",
+       ".pygments .vi { color: #19177C } /* Name.Variable.Instance */\n",
+       ".pygments .vm { color: #19177C } /* Name.Variable.Magic */\n",
+       ".pygments .il { color: #666666 } /* Literal.Number.Integer.Long */span.lineno {\n",
+       "    color: lightgray;\n",
+       "}\n",
+       "\n",
+       "</style>\n",
+       "<div class=\"pygments\"><pre><span></span><span class=\"lineno\">1 </span><span class=\"nn\">---</span>\n",
+       "<span class=\"lineno\">2 </span><span class=\"l l-Scalar l-Scalar-Plain\">num_workers</span><span class=\"p p-Indicator\">:</span> <span class=\"l l-Scalar l-Scalar-Plain\">100</span>\n",
+       "<span class=\"lineno\">3 </span><span class=\"l l-Scalar l-Scalar-Plain\">inventory</span><span class=\"p p-Indicator\">:</span> <span class=\"l l-Scalar l-Scalar-Plain\">brigade.plugins.inventory.simple.SimpleInventory</span>\n",
+       "<span class=\"lineno\">4 </span><span class=\"l l-Scalar l-Scalar-Plain\">SimpleInventory</span><span class=\"p p-Indicator\">:</span>\n",
+       "<span class=\"lineno\">5 </span>    <span class=\"l l-Scalar l-Scalar-Plain\">host_file</span><span class=\"p p-Indicator\">:</span> <span class=\"s\">&quot;../../../examples/inventory/hosts.yaml&quot;</span>\n",
+       "<span class=\"lineno\">6 </span>    <span class=\"l l-Scalar l-Scalar-Plain\">group_file</span><span class=\"p p-Indicator\">:</span> <span class=\"s\">&quot;../../../examples/inventory/groups.yaml&quot;</span>\n",
+       "</pre></div>\n",
+       "\n"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%highlight_file config.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-info\">\n",
+    "**Note:** You can also use environment variables.\n",
```

Comment:
Perhaps the mention of environment variables could be removed. I'm thinking mostly that the `SimpleInventory` arguments won't be possible to set using environment variables. Since that is included in the short config file above it might just confuse someone. Also the documentation for the config object shows that you can use environment variables.